config/secrets.yml

```diff
@@ -0,0 +1,8 @@
+development:
+  secret_key_base: '274b52c8af8dbcf6dca835e45b607244d76fd28501e873d477daaf1cbc450645434d1cf4b7935df3dcd859ae46a4dadcd26bd22eb506eed28fffefb688651989'
+
+test:
+  secret_key_base: '274b52c8af8dbcf6dca835e45b607244d76fd28501e873d477daaf1cbc450645434d1cf4b7935df3dcd859ae46a4dadcd26bd22eb506eed28fffefb688651989'
+
+production:
+  secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
```

Comment:
This will need to be populated from puppet and deployment, like we want to do the mongoid URL.